### ISSUE TITLE
Zoom increase/decrease buttons on either sides of the slider

### ISSUE THREAD

**Issue by anoopelias** [scottcheng/cropit#39]
Recently, I had to incorporate a crop tool in one of our apps, looked through a few and chose cropit. It has been working fantastic since. Thank you!

BTW our UX was of the opinion that I should add a '-' and '+' buttons on either side of the slider (Like you have on home page) or else it may not be intuitive to the user. I did that outside of the plugin, but now that I think about it, it is possible to incorporate it back to the plugin. I could work on a PR and get it out fairly easily. Do you think you would be interested?

**Comment by scottcheng** [comment]
Hey @anoopelias, great to hear that cropit is working well for you, and thanks for the proposal! That definitely sounds like a good idea, but one of the principles I have for cropit is that I wanted to keep it minimal, in particular not to have CSS in it, so the user of it can style it from scratch instead of overriding the existing ones. However in my opinion, different people would most likely want to style `-` and `+` in different ways, and give them different behaviors (e.g. different zoom steps). Therefore I've exposed simple APIs to get/set zoom, which I suppose you are using, to achieve such functionality, and while it will definitely be useful to some, I would like not to have extraneous options and complex design in cropit. Hope you understand!

**Comment by anoopelias** [comment]
@scottcheng While adding `-` and `+` I saw there are a few issues to be taken care of, apart from css. Like [this](http://stackoverflow.com/questions/28127507/incrementing-value-continuously-on-mouse-hold) for example. However I understand that you don't want to increase complexity in terms of zoom step configuration etc. I also suspected it might be overkill.

Off topic: My cropit based implementation went through QA on mobile devices (Android Chrome/Firefox) yesterday and it worked like a charm. :). Absolutely no change needed.
